### DISCUSSION TITLE
Sunbird ED Release Notes Template

### DISCUSSION THREAD

**Discussion by anisha147** [Sunbird-Ed/Community#616] [Documentation]
Hi All,

Please review and share your feedback and suggestions to improve the Sunbird ED release notes template. Could you share the input by 12-Sep-23?

Link: https://docs.google.com/document/d/1nZWIaKXKs6oHTIkHhGVgPY6Uex3aP3llTY9xMTHhRh0/edit

cc: [@Krishnaj20](https://github.com/Krishnaj20) [@gandham-santhosh](https://github.com/gandham-santhosh) [@chitranshu-keshav](https://github.com/chitranshu-keshav) [@kiranharidas187](https://github.com/kiranharidas187) [@Prateek-slokam](https://github.com/Prateek-slokam)
[@parthlawate](https://ekstep.slack.com/team/U0HB3HJ73)
 [@baljeetsingh](https://github.com/baljeetsingh) [@manojLondhe](https://github.com/manojLondhe) [@vinukumar-vs](https://github.com/vinukumar-vs) @swayangjit @rajeevsathish @PoojaJManjooran @ashokreddy1208 [@pallakartheekreddy](https://github.com/pallakartheekreddy) @sajeshkayyath @AmiableAnil
[@Mohit](https://ekstep.slack.com/team/U29EH3MSM) @rhwarrier @anandp504

**Comment by ashokreddy1208** [comment]
@Krishnaj20 @gandham-santhosh @chitranshu-keshav @kiranharidas187 @Prateek-slokam @parthlawate @baljeetsingh @manojLondhe @vinukumar-vs 

Can you please review ED release notes template and share your feedback as early as possible.
